windows/ReactTestApp/ReactInstance.cpp

```diff
@@ -0,0 +1,53 @@
+#include "pch.h"
+#include "ReactInstance.h"
+#include <filesystem>
+
+using namespace winrt::Microsoft::ReactNative;
+
+namespace ReactTestApp
+{
+    void ReactInstance::LoadJSBundleFrom(JSBundleSource source)
+    {
+        auto instanceSettings = reactNativeHost_.InstanceSettings();
```

Comment:
You need to make a reference otherwise you'll get a copy.
```suggestion
        auto &instanceSettings = reactNativeHost_.InstanceSettings();
```